app.py

```diff
@@ -0,0 +1,169 @@
+from flask import Flask, jsonify, request
+import os
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import Column, Integer, String, Float, Text
+from os import environ, path
+from dotenv import load_dotenv
+from db import db
+from ma import ma
+from flask_jwt_extended import JWTManager, jwt_required, create_access_token
+
+
+from models.user import User
+from models.recipe import Recipe
+from schemas.user import UserSchema, user_schema, users_schema
+from schemas.recipe import Recipe, recipe_schema, recipes_schema
+
+
+application = Flask(__name__)
+basedir = os.path.abspath(os.path.dirname(__file__))
+application.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///' + os.path.join(basedir, 'users.db')
+application.config['JWT_SECRET_KEY'] = environ.get('JWT_SECRET_KEY')
+
+
+db.init_app(application)
+ma.init_app(application)
+jwt = JWTManager(application)
+
+
+'''
+test data
+'''
+
+@application.cli.command('db_create')
+def db_create():
+    db.create_all()
+    print('DB created')
+
+
+@application.cli.command('db_drop')
+def db_drop():
+    db.drop_all()
+    print('DB dropped')
+
+
+@application.cli.command('db_seed')
+def db_seed():
+    fish_tacos = Recipe(recipe_name='Fish tacos', recipe_description='Delicios fish tacos from Tijuana', meal='Always', time=25, ingredients='fish and torillas', instructions='Fry everything together and hope for the best')
+    chicken_katsu = Recipe(recipe_name='Chicke Katsu', recipe_description='The best chicken katsu', meal='Always', time=25, ingredients='Chiken and katsu curry', instructions='order online from cocoro on delivroo, must me in Kentish')
+
+    db.session.add(fish_tacos)
+    db.session.add(chicken_katsu)
+
+    test_user = User(name='William', email='test@test.com', password='Password')
+
+    db.session.add(test_user)
+    db.session.commit()
+    print('Database seeded')
+
+
+@application.before_first_request
+def create_tables():
+    db.create_all()
+
+
+# Endpoints
```

Comment:
Rather than a comment, can the routes not go in their own file?

Reply:
yes, I need to move them to the resources folder, but I will do that for second round once I can get the pg database working, probs by the end of the week.